js/package.json

```diff
@@ -45,6 +45,7 @@
     "@bufbuild/protobuf": "^1.10.0",
     "@types/bytebuffer": "^5.0.49",
     "bitset": "^5.1.1",
-    "bytebuffer": "^5.0.1"
+    "bytebuffer": "^5.0.1",
+    "geojson": "^0.5.0"
```

Comment:
Can we do without this package? It looks like it is only being used for types but does not actually provide types and is old: https://www.npmjs.com/package/geojson